models/us_nature_gerda/code/upload.py

```
@@ -0,0 +1,99 @@
+        try:
+            st.delete_table(mode="staging", not_found_ok=True)
+        except Exception as e:
+            print(f"  [warn] staging cleanup: {e}")
+        tb.create(
+            path=path,
+            source_format="parquet",
+            if_table_exists="replace",
+            if_storage_data_exists="replace",
+            if_dataset_exists="pass",
+        )
+        q = f"select count(*) n from `{BILLING}.{a.dataset}_staging.{name}`"
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)